cogito/put_private_test.go

```diff
@@ -264,11 +292,13 @@ func TestParseGitPseudoURLFailure(t *testing.T) {
 			wantErr: "invalid git SSH URL git@github.com/Pix4D/cogito.git: want exactly one ':'",
 		},
 		{
+			// currently failing due to the new safeParse method returning only the error
 			name:    "invalid HTTPS URL",
 			inURL:   "https://github.com:Pix4D/cogito.git",
 			wantErr: `parse "https://github.com:Pix4D/cogito.git": invalid port ":Pix4D" after host`,
```

Comment:
Why not just fix the test?
```suggestion
			wantErr: `invalid port ":Pix4D" after host`,
```

Reply:
Actually because for now I don't really know if I should either fix the test to remove the URL, or adapt the method to keep some debug information